### PR TITLE
MP LRA Blog ID review

### PR DESCRIPTION
**Pull request by dmuelle**

### CODE REVIEW

File: posts/2020-12-18-microprofile-long-running-actions-beta.adoc [hunk 1]
```diff
@@ -77,7 +81,7 @@ In order to start a coordinator in Open Liberty you first need to enable the mpL
         <feature>mpLRACoordinator-1.0</feature>
     </featureManager>
    
-<!-- To access this server from a remote client add a host attribute to the following element, e.g. host="*" -->
+<!-- To access this server from a remote client, add a host attribute to the following element, for example, host="*" -->
```
Comment (yongja79) [inline posts/2020-12-18-microprofile-long-running-actions-beta.adoc]: This text should not be changed as it is what is in the server.xml by default

File: posts/2020-12-18-microprofile-long-running-actions-beta.adoc [hunk 2]
```diff
@@ -12,38 +12,42 @@ open-graph-image: https://openliberty.io/img/twitter_card.jpg
...
+The LRA model relies on having compensating actions for all business interactions and ensures that when the activity ends, all the work is either accepted or will be compensated. How each activity gets compensated depends on the individual services, but the model defines what triggers a compensating action and when they are executed.
 
 == LRA in Open Liberty
 
 In Open Liberty there are 2 parts to the using LRA:
 
-* LRA Participants which are applications using MicroProfile LRA annotations that cause them to become involved in the LRA
-* LRA Coordinator which manages the LRA processing. It handles the initialisation of the LRA, enlisting in a LRA and the completion or compensating of a LRA. This is enabled through standard Liberty configuration.
+* LRA participants, which are applications that use MicroProfile LRA annotations that involve them in the LRA
+* the LRA coordinator, which manages the LRA processing. It handles the initialization of the LRA, the enlistment of services in the LRA, and the completion or compensating of the LRA. The LRA Coordinator is enabled through standard Liberty configuration.
```
Comment (yongja79) [inline posts/2020-12-18-microprofile-long-running-actions-beta.adoc]: "LRA Coordinator" should be "LRA coordinator" to be consistent with the changes made

File: posts/2020-12-18-microprofile-long-running-actions-beta.adoc [hunk 3]
```diff
@@ -176,26 +182,28 @@ Finally the compensate method looks like this:
...
-While these 3 annotations form the basics of a LRA there are several more that can be used
+While these three annotations form the basics of an LRA, there are several more annotations avaialble:
 
-* `@Forget` - Called if the complete or compensate methods failed and you want to release any resources allocated to the LRA
-* `@Leave` - Called if the class is no longer interested in the LRA
-* `@Status` - When invoked a method with this annotation will return the status of the LRA
-* `@AfterLRA` - When a LRA is in its final state, the method with this annotation is called
+* `@Forget` - called if the complete or compensate methods fail and you want to release any resources that were allocated to the LRA
+* `@Leave` - called if the class is no longer interested in the LRA
+* `@Status` - When a method with this annotation is invoked, it returns the status of the LRA.
```
Comment (yongja79) [inline posts/2020-12-18-microprofile-long-running-actions-beta.adoc:191]: Sometimes added a `.` at the end of the point and other times not

**Review by yongja79 — CHANGES_REQUESTED** [review]

File: posts/2020-12-18-microprofile-long-running-actions-beta.adoc (hunk 1 above)
Comment (dmuelle) [reply posts/2020-12-18-microprofile-long-running-actions-beta.adoc]: ok- sorry thought it was a comment on the example- reverted it

File: posts/2020-12-18-microprofile-long-running-actions-beta.adoc (hunk 2 above)
Comment (dmuelle) [reply posts/2020-12-18-microprofile-long-running-actions-beta.adoc]: fixed throughout

File: posts/2020-12-18-microprofile-long-running-actions-beta.adoc (hunk 3 above)
Comment (dmuelle) [reply posts/2020-12-18-microprofile-long-running-actions-beta.adoc:191]: changed them all to be complete sentences that end with a period

**Review by yongja79 — APPROVED** [review]
LGTM

**Pull request merged**